_includes/amp.html

```diff
@@ -126,50 +131,37 @@
   custom-element="amp-position-observer"
   src="https://cdn.ampproject.org/v0/amp-position-observer-0.1.js"
 ></script>
-{% endif %}
-
-{% if page.amp_components contains 'amp-animation' %}
```

Comment:
@konarie1990 Beware your formatter changing pages and lines that aren't necessary to the pull request. It doesn't look like anything in this files changed, so we can put it back to how it was and the diff should go away